### PR TITLE
Login Page Frontend

### PR DESCRIPTION
**Pull request by leehoang2000**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Frontend Login Page

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: frontend/src/pages/LoginPage/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// import React from 'react'
```
Comment (alitonia) [inline frontend/src/pages/LoginPage/index.js:1]: As we have a linter already, consider running `yarn lint:fix` before commit to have correct identation

File: frontend/src/pages/LoginPage/LoginPage.styles.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+  /* White */
+
+  color: #ffffff;
+`;
+
+export const LoginSubText = styled.div`
+  /* Log in to use our services. Make sure your account has been created by your instructor */
+
+  position: absolute;
```
Comment (alitonia) [inline frontend/src/pages/LoginPage/LoginPage.styles.js:72]: Please check these css again, it is bugged on my computer ( width 1280). Try to make the page responsive on at least 3 size (360, 720, 1080)

**Review by alitonia — CHANGES_REQUESTED** [review]
Please change the css parts to be responsive, then run the linter to make your code more glamorous before commit.

File: frontend/src/pages/LoginPage/LoginPage.styles.js (hunk 2 above)
Comment (leehoang2000) [reply frontend/src/pages/LoginPage/LoginPage.styles.js]: Text overflowing has been resolved. Responsive on 3 screen sizes will require explicit design.

File: frontend/src/pages/LoginPage/index.js (hunk 1 above)
Comment (leehoang2000) [reply frontend/src/pages/LoginPage/index.js]: yarn lint:fix and yarn prettier:fix . has been run before commit

**Review by alitonia — APPROVED** [review]
Change accepted. Responsive is delayed till later design details

**Pull request merged**